presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardCleaner.java

```diff
@@ -330,10 +361,43 @@ void deleteOldCompletedTransactions()
     }
 
     @VisibleForTesting
+    synchronized Set<UUID> getLocalShards()
+    {
+        return storageService.getStorageShards();
+    }
+
+    @VisibleForTesting
+    synchronized void cleanLocalShardsAtStartup(Set<UUID> local)
```

Comment:
Let's rename this and the log message since it's used for JMX. How about "immediately" instead of "at startup".